scripts/combine_labels.py

```diff
@@ -101,6 +106,12 @@ def main(args):
                       labelers for a record to be considered anomalous (should \
                       be a number between 0 and 1)")
   
+  parser.add_argument("--windowSize",
```

Comment:
I think we should make this a constant in the code. This is not something a NAB user would be setting.